UI/dashboard.html

```diff
@@ -0,0 +1,86 @@
+                                    <div class="pie-wrapper progress-50 ">
+                                            <span class="label">50<span class="smaller">%</span></span>
+                                            <div class="pie">
+                                            <div class="left-side half-circle"></div>
+                                            <div class="right-side half-circle"></div>
+                                            </div>
+                                            <div class="shadow"></div>
+                                    </div>
+                                    <span class="text">Loan Repayed</span>
+                                </div>
+                                <div class="metric">
+                                    <div class="pie-wrapper progress-50 ">
+                                            <span class="label">50<span class="smaller">%</span></span>
+                                            <div class="pie">
+                                            <div class="left-side half-circle"></div>
+                                            <div class="right-side half-circle"></div>
+                                            </div>
+                                            <div class="shadow"></div>
+                                    </div>
+                                    <span class="text">User Verified</span>
+                                </div>
+                        </div>
+                    </div>
+                        <div class="generic-card metric-card">
+                            <i class="fas fa-money-bill-wave"></i>
+                            <span>2</span>
+                            <h3 class="title">Pending Repayments</h3>
+                        </div>   
+                        <div class="generic-card metric-card">
+                            <i class="fas fa-users"></i>
+                            <span>2</span>
+                            <h3 class="title">New Users</h3>
+                        </div>        
+                </div>
+ 
+                <div class="content">
+                    <div class="page-title">
+                        <span>New Loan Applications</span>
+                    </div>
+                    
+                    <div class="card-container" id = "card-container-root"></div>  
+                    <div class="page-title">
+                            <span>New Clients</span>
+                        </div>
+                        <div class="card-container" id = "card-container-root"></div>                  
+                </div>
+            </div>
+        </div>
+        <div class="overlay full-overlay">
+                <i class="close-btn icon close"></i>
+                <div id="single-loan-root">
+
+                </div>
+        </div>
+
+    </div>    
+</body>
+<script src="assets/js/quick-credit.js" id="main-js"></script>
+</html>
```

Comment:
GitHub requires you to add a `new line` at the end of each file. Please fix this and other occurrences 😄